java-support/src/main/scala/io/cloudstate/javasupport/impl/EntityDiscoveryImpl.scala

```diff
@@ -25,11 +25,13 @@ import io.cloudstate.javasupport.{BuildInfo, StatefulService}
 
 class EntityDiscoveryImpl(system: ActorSystem, services: Map[String, StatefulService]) extends EntityDiscovery {
 
+  private val config = system.settings.config.getConfig("cloudstate")
+
   private val serviceInfo = ServiceInfo(
     serviceRuntime = sys.props.getOrElse("java.runtime.name", "") + " " + sys.props.getOrElse("java.runtime.version",
                                                                                               ""),
-    supportLibraryName = BuildInfo.name,
-    supportLibraryVersion = BuildInfo.version
+    supportLibraryName = config.getString("library.name"),
+    supportLibraryVersion = config.getString("library.version")
```

Comment:
Does it matter that this no longer uses the automatically generated current version for cloudstate-java-support, but is instead fixed in config?

Reply:
@pvlugter  This exists for another support idiom library (like Kotlin) that uses lib java-support as a base and defines a hocon file reference.conf or application.conf and can overwrite the version of the java library with its own information.

Reply:
Sure, understand the usage. But currently published versions of cloudstate-java-support, like 0.4.7, will report their version as 0.4.3 since it's hardcoded in the config. What we want is for the name and version to be overridable to config, but use the generated build info by default. I'll make an update for this.

Reply:
Updated in #294